### PR TITLE
[WIP] Unit test with random seed failure

### PR DESCRIPTION
**Pull request by KexinFeng**
This PR tries to enable the unit test for the sampling autoregressive searching. The assertion is true when locally testing it. But in the PR continueous test, the assertion becomes false.

Update:
---
Now the assertion in the unit test passes the PR test. However, local test will fail at that assertion.

So the issue is, after setting the random seed, the outputs, even though fixed, are different between local testing and PR testing.

### CODE REVIEW

**Review by lanking520 — CHANGES_REQUESTED** [review]
we removed those PT files since it violates the Jacoco test rules

**Review by lanking520 — APPROVED** [review]

File: engines/python/setup/djl_python/tests/test_scheduler.py [hunk 1]
```diff
@@ -156,15 +161,20 @@ def test_sampling_scheduler(self):
...
         results = scheduler.collect_results()
 
-        # assert tokenizer.decode(results[1][:30]) == "When your legs don't work like they used to before And I can't " \
-        #                                             "sweep you off your feet, you're right, I'm done for the"
-        assert tokenizer.decode(results[2][:30]) == "There's a time that I remember, when I did not know what to do " \
-                                                    "with my life. I was in a very bad mood. I was"
-        # assert tokenizer.decode(results[0][:30]) == "Memories follow me left and right. I can't help but feel that " \
-        #                                             "I've been given a chance to do something different. I've been told"
-
+        print('=====++++++++++==============')
```
Comment (frankfliu) [inline engines/python/setup/djl_python/tests/test_scheduler.py:164]: remove this line

File: engines/python/setup/djl_python/tests/test_scheduler.py [hunk 2]
```diff
@@ -156,15 +161,20 @@ def test_sampling_scheduler(self):
...
+        assert tokenizer.decode(results[0][:30]) == "Memories follow me left and right. I can't remember the last " \
+                                                    "time I had to do this, and I'm glad that I did.\n"
+        assert tokenizer.decode(
+            results[1][:30]
+        ) == """When your legs don't work like they used to before And I can't sweep you off your face" one female announced... " Everyone seems"""
+        assert tokenizer.decode(
+            results[2][:30]
+        ) == "There's a time that I remember, when I did not know what to do with my life. I was in a very bad mood. I was"
+
+        print('======+++++++++++++=========')
```
Comment (frankfliu) [inline engines/python/setup/djl_python/tests/test_scheduler.py:177]:
```suggestion
```

File: engines/python/setup/djl_python/scheduler/seq_batch_scheduler.py [hunk 3]
```diff
@@ -43,8 +43,8 @@ def __init__(self, lm_block: LMBlock, default_search_algorithm: str,
         self.results: Dict[int, List[int]] = defaultdict(list)
 
         self.seq_batchers: Dict[
-                           Type[SeqBatcher]:List[SeqBatcher]] = defaultdict(
-            list)  # {key: List[SeqBatcher]}
+            Type[SeqBatcher]:List[SeqBatcher]] = defaultdict(
+                list)  # {key: List[SeqBatcher]}
```
Comment (frankfliu) [inline engines/python/setup/djl_python/scheduler/seq_batch_scheduler.py:47]: remove comments part

File: engines/python/setup/djl_python/tests/test_scheduler.py [hunk 4]
```diff
@@ -156,15 +161,20 @@ def test_sampling_scheduler(self):
...
-        #                                             "sweep you off your feet, you're right, I'm done for the"
-        assert tokenizer.decode(results[2][:30]) == "There's a time that I remember, when I did not know what to do " \
-                                                    "with my life. I was in a very bad mood. I was"
-        # assert tokenizer.decode(results[0][:30]) == "Memories follow me left and right. I can't help but feel that " \
-        #                                             "I've been given a chance to do something different. I've been told"
-
+        print('=====++++++++++==============')
         for i, ret in results.items():
-            print('\n{}:'.format(i), tokenizer.decode(ret))
+            print('\n{}:'.format(i), tokenizer.decode(ret[:30]))
```
Comment (frankfliu) [inline engines/python/setup/djl_python/tests/test_scheduler.py:166]: 1. Avoid using print, use logger if needed
2. Do we really need this log in production?

File: engines/python/setup/djl_python/scheduler/seq_batch_scheduler.py [hunk 5]
```diff
@@ -55,7 +55,8 @@ def add_request(self,
                     search_algorithm: str = None,
                     search_configs: List[SearchConfig] = None,
                     kv_cache: Union[Tuple, None] = None,
-                    kv_cache_prompt_ids: Union[Dict[int, torch.tensor], None] = None):
+                    kv_cache_prompt_ids: Union[Dict[int, torch.tensor],
```
Comment (frankfliu) [inline engines/python/setup/djl_python/scheduler/seq_batch_scheduler.py:58]: In future, please create a PR that only do the format. It will be easy for others to review the code. It's not clear what exactly this PR is doing

**Review by frankfliu — CHANGES_REQUESTED** [review]

File: engines/python/setup/djl_python/tests/test_scheduler.py (hunk 4 above)
Comment (lanking520) [reply engines/python/setup/djl_python/tests/test_scheduler.py:166]: this is test code

File: engines/python/setup/djl_python/tests/test_scheduler.py (hunk 1 above)
Comment (lanking520) [reply engines/python/setup/djl_python/tests/test_scheduler.py:164]: Those are test code, I think it's ok

**Review by lanking520 — APPROVED** [review]

**Review by frankfliu — APPROVED** [review]
